Documentation/helm-values.rst

```diff
@@ -1357,7 +1357,7 @@
      - bool
      - ``false``
    * - :spelling:ignore:`hubble.metrics.enabled`
-     - Configures the list of metrics to collect. If empty or null, metrics are disabled. Example:    enabled:   - dns:query;ignoreAAAA   - drop   - tcp   - flow   - icmp   - http  You can specify the list of metrics from the helm CLI:    --set hubble.metrics.enabled="{dns:query;ignoreAAAA,drop,tcp,flow,icmp,http}"
+     - Configures the list of metrics to collect. If empty or null, metrics are disabled. Example:    enabled:   - dns:query;ignoreAAAA   - drop   - tcp   - flow   - icmp   - http  You can specify the list of metrics from the helm CLI:    --set metrics.enabled="{dns:query;ignoreAAAA,drop,tcp,flow,icmp,http}"
```

Comment:
That was kind of irrelevant "fixup" but anyways we can fix it in a different standalone PR.

Reply:
Ah, yes.